### PR TITLE
Fixed adding ticket, added get-tickets endpoint

### PR DESCRIPTION
**Pull request by kujawad**

### CODE REVIEW

File: src/main/java/com/metabug/persistence/dao/TicketRepository.java [hunk 1]
```diff
@@ -4,9 +4,12 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
 import java.util.UUID;
 
 @Repository
 public interface TicketRepository extends JpaRepository<Ticket, UUID> {
     Ticket findByTitle(final String title);
```
Comment (krzjez) [inline src/main/java/com/metabug/persistence/dao/TicketRepository.java]: what if 2 tickets have the same title? Maybe it should be a list or findByID?

File: src/main/java/com/metabug/service/TicketService.java [hunk 2]
```diff
@@ -18,22 +21,18 @@ public TicketService(final TicketRepository ticketRepository,
...
-        final String author = ticketDto.getAuthor();
-
-        if (userService.findUserByLogin(author) != null) {
-            ticket.setAuthorId(userService.findUserByLogin(author).getId());
-        } else {
-            ticket.setAuthorId(userService.findUserByEmail(author).getId());
-        }
+        //TODO: check if users exists? just in case?
+        ticket.setAuthorId(userService.findUserByLogin(principal.getName())
+                .getId());
```
Comment (krzjez) [inline src/main/java/com/metabug/service/TicketService.java]: .getId one space to the left, so it aligns to the .setAuthorId

File: src/main/java/com/metabug/web/controller/TicketController.java [hunk 3]
```diff
@@ -29,16 +39,26 @@ public ModelAndView getView() {
 
         modelAndView.setViewName("add-ticket");
         modelAndView.addObject("ticketDto", ticketDto);
+        modelAndView.addObject("assignees", userService.findAll());
```
Comment (krzjez) [inline src/main/java/com/metabug/web/controller/TicketController.java]: why assignees and not developers to keep it consistent with what we name it in DB?

**Review by krzjez — CHANGES_REQUESTED** [review]

File: src/main/java/com/metabug/persistence/dao/TicketRepository.java (hunk 1 above)
Comment (kujawad) [reply src/main/java/com/metabug/persistence/dao/TicketRepository.java]: Changed to `findAllByTitle`, but still it's not used anywhere, for now

File: src/main/java/com/metabug/service/TicketService.java (hunk 2 above)
Comment (kujawad) [reply src/main/java/com/metabug/service/TicketService.java]: Formatter, aligned to one-line

File: src/main/java/com/metabug/web/controller/TicketController.java (hunk 3 above)
Comment (kujawad) [reply src/main/java/com/metabug/web/controller/TicketController.java]: Removed, because we're not setting assignee in `/add-ticket`

**Review by krzjez — APPROVED** [review]

**Pull request merged**